pkgs/by-name/um/umu-launcher/package.nix

```diff
@@ -2,13 +2,21 @@
   buildFHSEnv,
   lib,
   umu-launcher-unwrapped,
+  extraPkgs ? pkgs: [ ],
+  extraLibraries ? pkgs: [ ],
 }:
 buildFHSEnv {
   pname = "umu-launcher";
   inherit (umu-launcher-unwrapped) version meta;
 
-  # Use umu-launcher-unwrapped from the package args, to simplify overriding
-  targetPkgs = _: [ umu-launcher-unwrapped ];
+  targetPkgs =
+    pkgs:
+    [
+      # Use umu-launcher-unwrapped from the package args, to support overriding
+      umu-launcher-unwrapped
+    ]
+    ++ extraPkgs pkgs;
```

Comment:
Wow nixfmt has made that ugly :boom: :fire_engine: 

IDK if this is still the case, but it used to format things nicer if you made the list-literal the last part of the concatenation:

```suggestion
  targetPkgs = pkgs: extraPkgs pkgs ++ [
    # Use umu-launcher-unwrapped from the package args, to support overriding
    umu-launcher-unwrapped
  ];
```

No big deal, either way :slightly_smiling_face: